src/frontend/next/src/components/Login/index.tsx

```diff
@@ -0,0 +1,57 @@
+import Link from 'next/link';
+import { ListItem } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import LoggedIn from './LoggedIn';
+import LoggedOut from './LoggedOut';
+
+import config from '../../config';
+import { useUser } from '../UserProvider';
+
+/**
+ * Show either a Login button (if user isn't authenticated)
+ * or a welcome message and Logout button.
+ */
+
+const useStyles = makeStyles((theme) => ({
+  item: {
+    color: theme.palette.background.default,
+    fontFamily: 'Roboto, sans-serif',
+    textDecoration: 'none',
+    fontSize: '1.5rem',
+    justifyContent: 'center',
+    fontWeight: 500,
+    lineHeight: 1.75,
+  },
+}));
+
+type LoginProps = {
+  // style is a class name passed from MobileHeader component, so it need to be a string
+  isMobile: boolean | undefined;
+};
+
+const Login = ({ isMobile = false }: LoginProps) => {
+  const { isLoggedIn } = useUser();
+  const classes = useStyles();
+  // Check if using mobile header which means a style prop exists
```

Comment:
You can remove this comment completely.

Reply:
@humphd : finished!
Can you approve again?